spec/features/bars/administrator_can_delete_bar_spec.rb

```diff
@@ -35,4 +36,46 @@
 
     expect(page).to_not have_content("Delete")
   end
+
+  scenario 'bar is no longer in database' do
+    admin = FactoryGirl.create(:user, admin: true)
+    bar = FactoryGirl.create(:bar)
+
+    sign_in_as(admin)
+
+    visit bar_path(bar)
+
+    click_link("Delete")
+
+    expect(page).to_not have_content(bar.name)
+  end
+
+  scenario 'bar details page no longer exists' do
+    admin = FactoryGirl.create(:user, admin: true)
+    bar = FactoryGirl.create(:bar)
+
+    sign_in_as(admin)
+
+    visit bar_path(bar)
+
+    click_link("Delete")
+
+    expect(page).to_not have_content(bar.name)
+    expect(page).to_not have_content(bar.address)
+    expect(page).to_not have_content(bar.state)
+  end
+
+  scenario 'reviews of this bar no longer exist' do
+    admin = FactoryGirl.create(:user, admin: true)
+    review = FactoryGirl.create(:review)
+
+    sign_in_as(admin)
+
+    visit bar_path(review.bar)
+
+    click_link("Delete", match: :first)
+
+    expect(page).to_not have_content(review.title)
+    expect(page).to_not have_content(review.body)
```

Comment:
This is is also an incorrect test. Trace back to where your redirect is on a successful bar deletion. Again, there would be no bar details page after the delete, so there would be no reviews. Your test might pass, but that's because the page you are redirected to doesn't have `review.title` or `review.body` which it wouldn't have anyways.